src/Microsoft.DotNet.Archive/LZMA/ICoder.cs

```diff
@@ -0,0 +1,157 @@
+// ICoder.h
```

Comment:
Yeah, we engaged legal a week or two ago when I started this.  They said we were fine to use it.  Just working out the details of copyright/attribution now.